test/TestExtensions/DefaultClusterFixture.cs

```diff
@@ -22,7 +22,6 @@ public ISiloHostBuilder CreateSiloBuilder(string siloName, ClusterConfiguration
                 return new SiloHostBuilder()
                     .ConfigureSiloName(siloName)
                     .UseConfiguration(clusterConfiguration)
-                    .UseGrainBasedMembership()
```

Comment:
if we are removing this, then we should add a test to configure GrainBasedMemebership in the new way. Otherwise there's no test using testing `UseGrainBasedMembership`

Reply:
Yes, I'm aware of that, and agree, but the soon to come PR that revamps test cluster will be using it extensively